Documentation/ReleaseNotes/PreviewReleases/Release-Notes-v1.0.0-preview.8.md

```diff
@@ -0,0 +1,106 @@
+<h1 align="center" style='color:mediumseagreen;font-weight:bold'>Velaptor Preview Release Notes - v1.0.0-preview.8</h1>
+
+<h2 align="center" style='font-weight:bold'>Quick Reminder</h2>
+
+As with all software, including releases, there is always a chance for issues and bugs.  It is also common to miss changes in the release notes when there are many.  This is even more common in preview releases.
+
+---
+
+<h2 style="font-weight:bold" align="center">New 🎉</h2>
+
+1. Added new constructor to `CornerRadius` struct for convenience.  
+   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * This make it easier to set all of the radius values to the same value.
+2. Added a new property named `HalfWidth` to the `RectShape` struct.  
+   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+3. Added a new property named `HalfHeight` to the `RectShape` struct.  
+   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+4. Added a new property named `Top` to the `RectShape` struct to get or set the location of the top of the rectangle.  
+   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * This will automatically update the `Position` property of the rectangle.
+5. Added a new property named `Right` to the `RectShape` struct to get or set the location of the top of the rectangle.  
+   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * This will automatically update the `Position` property of the rectangle.
+6. Added a new property named `Bottom` to the `RectShape` struct to get or set the location of the top of the rectangle.  
+   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * This will automatically update the `Position` property of the rectangle.
+1.  Added a new property named `Left` to the `RectShape` struct to get or set the location of the top of the rectangle.  
+    GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
+    * This will automatically update the `Position` property of the rectangle.
```

Comment:
Location of properties is incorrect in the text.  Make sure line numbering is sequential throughout the whole file.
```suggestion
5. Added a new property named `Right` to the `RectShape` struct to get or set the location of the right of the rectangle.  
   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
    * This will automatically update the `Position` property of the rectangle.
6. Added a new property named `Bottom` to the `RectShape` struct to get or set the location of the bottom of the rectangle.  
   GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
    * This will automatically update the `Position` property of the rectangle.
1.  Added a new property named `Left` to the `RectShape` struct to get or set the location of the left of the rectangle.  
    GitHub Issues: [#196](https://github.com/KinsonDigital/Velaptor/issues/196)
    * This will automatically update the `Position` property of the rectangle.
```